packages/esbuild/esbuild_packages.bzl

```diff
@@ -0,0 +1,70 @@
+"""Info for the esbuild packages used"""
+
+### These values are updated automaticly via `yarn update-esbuild-versions`
+_VERSION = "0.12.5"
+_DARWIN_AMD64_SHA = "6037c279c2bd09ae6ff490f18ea95ab514db044df63db47268411dfc48a49b16"
+_DARWIN_ARM64_SHA = "bb950e084302652a11c5d38aeae773fa6f46b864e7b89264c9354222aa8d326d"
+_LINUX_AMD64_SHA = "ea9bae194f4d18297ccb00139c62326e3da40835bbadbe21848c3e2cded35a3a"
+_LINUX_ARM64_SHA = "f70fbe27a5dab5dd6f19f76f60660ad59b8a6883652bb1fd878078cf922915ad"
+_WINDOWS_AMD64_SHA = "ce16dc8d565f8b30b538496961ba17c1977ddd400d0f20b7e89482ecd0944d5d"
+
+ESBUILD_PACKAGES = struct(
+    version = _VERSION,
+    platforms = dict({
+        "darwin_amd64": struct(
+            sha = _DARWIN_AMD64_SHA,
+            urls = [
+                "https://registry.npmjs.org/esbuild-darwin-64/-/esbuild-darwin-64-%s.tgz" % _VERSION,
+            ],
+            binary_path = "bin/esbuild",
+            exec_compatible_with = [
+                "@platforms//os:macos",
+                "@platforms//cpu:x86_64",
+            ],
+        ),
+        "darwin_arm64": struct(
+            sha = _DARWIN_ARM64_SHA,
+            urls = [
+                "https://registry.npmjs.org/esbuild-darwin-arm64/-/esbuild-darwin-arm64-%s.tgz" % _VERSION,
+            ],
+            binary_path = "bin/esbuild",
+            exec_compatible_with = [
+                "@platforms//os:macos",
+                "@platforms//cpu:arm64",
+            ],
+        ),
+        "linux_amd64": struct(
+            sha = _LINUX_AMD64_SHA,
+            urls = [
+                "https://registry.npmjs.org/esbuild-linux-64/-/esbuild-linux-64-%s.tgz" % _VERSION,
+            ],
+            binary_path = "bin/esbuild",
+            exec_compatible_with = [
+                "@platforms//os:linux",
+                "@platforms//cpu:x86_64",
+            ],
+        ),
+        "linux_arm64": struct(
+            sha = _LINUX_ARM64_SHA,
+            urls = [
+                "https://registry.npmjs.org/esbuild-linux-arm64/-/esbuild-linux-arm64-%s.tgz" % _VERSION,
+            ],
+            binary_path = "bin/esbuild",
+            exec_compatible_with = [
+                "@platforms//os:linux",
+                "@platforms//cpu:x86_64",
```

Comment:
I think this needs to be aarch64. I had a user report that arm64 did not work as an alias in these constraints - not sure if that also applies to the darwin_amd64 build as well.

Reply:
I put the wrong one for linux here (copy paste). Does `arm64` work? I don't have a machine to test this.

Reply:
Me either :-) But the user reported arm64 did not work: https://github.com/ankitects/esbuild_toolchain/pull/2

Reply:
Oh intersting, I see, thanks